src/pages/index.js

```diff
@@ -60,66 +107,149 @@ const editCardPopup = new PopupWithForm(
 );
 editCardPopup.setEventListeners();
 
+const editAvatarPopup = new PopupWithForm(
+  "#edit-avatar-modal",
+  (evt, formValues) => {
+    evt.preventDefault();
+
+    editAvatarPopup.renderLoading(false);
```

Comment:
`renderLoading(false)` is needed in `finally` to stop the loading. You need `renderLoading(true)` before the request to show `Saving...` in the button

**Please, correct it everywhere in the code**